### ISSUE TITLE
font-patcher DeprecationWarning with Python 3.8.5

### ISSUE THREAD

**Issue by mtsolmn** [ryanoasis/nerd-fonts#507]
#### 🎯 Subject of the issue

I get the following error when running `font-patcher` with the following command (using python3.8):
```
~/nerd-fonts > fontforge -script font-patcher '.../Input-Regular.ttf' --custom '<path-to-font>/new-icons.tff'
Copyright (c) 2000-2020. See AUTHORS for Contributors.
 License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
 with many parts BSD <http://fontforge.org/license.html>. Please read LICENSE.
 Version: 20200314
 Based on sources from 2020-05-20 15:35 UTC-D.
The glyph named Tcommaaccent is mapped to U+021A.
But its name indicates it should be mapped to U+0162.
The glyph named Delta is mapped to U+0394.
But its name indicates it should be mapped to U+2206.
The glyph named Omega is mapped to U+03A9.
But its name indicates it should be mapped to U+2126.
The glyph named tcommaaccent is mapped to U+021B.
But its name indicates it should be mapped to U+0163.
The glyph named mu is mapped to U+03BC.
But its name indicates it should be mapped to U+00B5.
Adding 54 Glyphs from Seti-UI + Custom Set
╢██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░╟ 4%font-patcher:823: DeprecationWarning: an integer is required (got type float).  Implicit conversion to integers using __int__ is deprecated, and may be removed in a future version of Python.
  glyph.left_side_bearing = 0.0
╢████████████████████████████████████████╟ 100%
```

Simply editing line 823 and 825 of `font-patcher` with `glyph.left_side_bearing = 0` and `glyph.right_side_bearing = 0`, respectively, solves the issue. However, I'm not sure if this is appropriate as I'm not familiar with the glyph formatting this modifies.
As stated in the title, I confirmed the script is using the Python 3.8.5 version on my system.

#### 🔧 Your Setup

- _Which font are you using: NA -> `font-patcher` issue_
- _Which terminal emulator are you using: alacritty_
- _Which operating system are you using: macOS Catalina_

**Comment by Antyos** [comment]
I'm getting the same warning using the Docker patcher.

Docker **nerdfonts/patcher** image ID: 92100385c0d4

Here's a file that will generate the warning: [CascadiaMonoPL-SemiLight.zip](https://github.com/ryanoasis/nerd-fonts/files/7026436/CascadiaMonoPL-SemiLight.zip). (Note: I had to zip the `.ttf` because GitHub does not officially support sharing them).

Command:
```bash
docker run -v "cascadia-code\build\ttf\static:/in" -v "cascadia-code\build\ttf\patched:/out" nerdfonts/patcher -c --careful -q
```

**Comment by ryanoasis** [comment]
Those lines have since been fixed @mtsolmn in #471

@Antyos the docker build is now working again and the latest docker image should work https://hub.docker.com/r/nerdfonts/patcher/tags

please re-open if that's not the case. thanks!
